src/proact/core.cljs

```diff
@@ -1,8 +1,17 @@
 (ns proact.core
   (:require [clojure.browser.repl :as repl]))
-
 ;; (repl/connect "http://localhost:9000/repl")
 
-(enable-console-print!)
+(defn replace_match [match context]
+    ;; eval a match and then if it works return it
+    (def res (.replace match (get (keyword (.split match #"\.") ) context)))
+    res
+)
 
-(println "Hello world!")
+(defn kirtify [vars context]
```

Comment:
@agamdua noted, although be aware I see this as an attempt to devalue my contributions to the greatest templating engine in existence